docs/advanced-table-feature.md

```diff
@@ -0,0 +1,65 @@
+#### Partitioned table
+
+Currently, Gravitino supports the following partitioning strategies:
+
+| Partitioning strategy | Json                                                | Java                           | SQL syntax                 | Description                                                                                                                 |
+|-----------------------|-----------------------------------------------------|--------------------------------|----------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Identity              | `{"strategy":"identity","fieldName":["score"]}`     | `Transforms.identity("score")` | `PARTITION BY score`       | Partition by a field or reference                                                                                           |
+| function              | `{"strategy":"functionName","fieldName":["score"]}` | `Transforms.hour("score")`     | `PARTITION BY hour(score)` | Partition by a function, currently, we support currently function, hour, year, day, bucket, month, truncate, list and range |
```

Comment:
Why do we choose use upper letter for the first letter?

Reply:
I'm not sure if it's considered conventional. What's your opinion?

Reply:
First, we should keep consistent. `function` uses `f` as the start. `Identify` uses `I`  as the start.
Second, could you refer to the document of Iceberg https://iceberg.apache.org/spec/#partition-transforms

Reply:
> Second, could you refer to the document of Iceberg [iceberg.apache.org/spec/#partition-transforms](https://iceberg.apache.org/spec/#partition-transforms)

I think we should provide a general description of them, not just limit them to Icebergs. Icebergs don't fully support this features.

Reply:
I mean the format of document. such as source type, etc.